### PR TITLE
sync development with main for most up to date code

### PR DESCRIPTION
**Pull request by Nkdiaz**
Changes:
1- re-add comment about leading 0s to config-example.ini
2- update reports.ipynb to have the severity table
3- removed unused requirements.txt and consolidate to parent requirements.txt
4- add google analytics
5- added severity table validation to the report.html to solved the error in main created by last main merge

Fixes:
https://github.com/cal-itp/data-infra/issues/963
https://github.com/cal-itp/reports/issues/106

Type of change:  new feature

How has it been tested: tested locally on my machine.

### CODE REVIEW

File: templates/report.html.jinja [hunk 1]
```diff
@@ -116,13 +116,50 @@ Monthly GTFS Quality Reports: {{parameters["DATE_MONTH_YEAR"]}} {{parameters["AG
                 {% endfor %}
             </tbody>
 
+            <!-- validation severity errors -->
+
+            {% if validation_severity is not defined %}
+
```
Comment (themightychris) [inline templates/report.html.jinja:122]: when does this case happen and is totally blank the right answer? shouldn't this case lead to the "Not available" message under the else?

Comment (lauriemerrell) [reply templates/report.html.jinja:122]: This is correct per a conversation between me, Andrew, and Natalya last week -- this is a new table that uses new data that wasn't defined for prior months. The condition is constructed such that all reports prior to March 2022 do not have their appearance altered (they don't have a message added to preserve their original format)

**Review by blakeflei — CHANGES_REQUESTED** [review]
Can we follow the PR template outlined in the [best practices doc](https://docs.google.com/document/d/1UbrGHAeVJM8c8SkDTOrcq1l373cz4Kcd1w_oDihjbg0/edit#)? 
More specifically:
- Can you please provide a link in the PR description to include: [https://github.com/cal-itp/data-infra/issues/963](https://github.com/cal-itp/data-infra/issues/963)
- Add the type of change: new feature

Also:
- Can we remove `reports/requirements.txt` in preference to `requirements.txt` in the root directory? More than one requirements.txt is confusing and should be avoided wherever possible.
- Is the inclusion of [pybigquery==0.7.0](https://github.com/cal-itp/reports/blob/2ba037ff4aab6ebdef80497e785108b6abaa1dc0/reports/requirements.txt#L90) intentional? 
- Should we include March and Feb 2022 in the [run_all_months.py](https://github.com/cal-itp/reports/blob/2ba037ff4aab6ebdef80497e785108b6abaa1dc0/reports/run_all_months.py#L17)?

**Review by blakeflei — CHANGES_REQUESTED** [review]
One last note in the requirements.txt file:
- calitp-py==0.0.6 still has the pybigquery dependency when building from this requirements.txt. Upping to the current version 0.0.15 should resolve
- Can also remove the empty space at the end of the requirements.txt

**Review by blakeflei — APPROVED** [review]

**Pull request merged**